src/STW.SubmissionApi.Presentation/STW.SubmissionApi.Presentation.csproj

```diff
@@ -1,7 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
     <ItemGroup>
+        <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.18.0-beta.1" />
```

Comment:
Is there a non-BETA version that's stable? Using Beta seems odd

Reply:
fixed to latest stable version